linux-tools/passwd/passwd.sh

```diff
@@ -61,48 +64,61 @@ function test01()
 
 function test02()
 {
-    tc_register "passwd --maximum"
-    passwd --maximum=90 $TC_TEMP_USER 1>$stdout 2>$stderr
-    tc_pass_or_fail $? "Setting maximum password lifetime to 90 Days failed"
+    grep -i "ubuntu" /etc/*-release >/dev/null 2>&1
+    if [ $? -ne 0 ];then  # Start of OS check
+    	tc_register "passwd --maximum"
+    	passwd --maximum=90 $TC_TEMP_USER 1>$stdout 2>$stderr
+    	tc_pass_or_fail $? "Setting maximum password lifetime to 90 Days failed"
 
-    tc_register "passwd --minimum"
-    passwd --minimum=30 $TC_TEMP_USER 1>$stdout 2>$stderr
-    tc_pass_or_fail $? "Setting minimum password lifetime to 30 days failed"
+    	tc_register "passwd --minimum"
+    	passwd --minimum=30 $TC_TEMP_USER 1>$stdout 2>$stderr
+    	tc_pass_or_fail $? "Setting minimum password lifetime to 30 days failed"
 
-    tc_register "passwd --warning"
-    passwd --warning=9 $TC_TEMP_USER 1>$stdout 2>$stderr
-    tc_pass_or_fail $? "settings to warn user before 9 days of password expiration failed"
+    	tc_register "passwd --warning"
+    	passwd --warning=9 $TC_TEMP_USER 1>$stdout 2>$stderr
+    	tc_pass_or_fail $? "settings to warn user before 9 days of password expiration failed"
+    fi
```

Comment:
In ubuntu, passwd has different option for all above tests
-x, --maxdays MAX_DAYS
-n, --mindays MIN_DAYS
-w, --warndays WARN_DAYS

They have to be handled for ubuntu as well